app-editors/scite/scite-3.5.5.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="5"
+EAPI="6"
```

Comment:
Please avoid doing this but instead perform a revision bump.

Reply:
(Why are you touching 3.5.5 in the first place? This change seems rather useless)